packages/lesmis-server/src/connections/MeditrakConnection.ts

```diff
@@ -30,4 +30,24 @@ export class MeditrakConnection extends SessionHandlingApiConnection {
   registerUser(userData: RequestBody) {
     return this.post('user', {}, userData);
   }
+
+  async updateUserEntityPermissions(data: RequestBody) {
+    // @ts-ignore
+    const { userEntityPermissionId, permissionGroupName } = data;
+
+    // get permission_group byName
+    const permissionGroups = await this.get('permissionGroups', {
+      filter: JSON.stringify({
+        name: permissionGroupName,
+      }),
+    });
+
+    return this.put(
+      `userEntityPermissions/${userEntityPermissionId}`,
```

Comment:
I believe `Tupaia Admin Panel` is required for this endpoint, how are we dealing with that for LESMIS users?